### DISCUSSION TITLE
LiFePO4 Batterie: gibt es PGNs für Einzelzellen-Status?

### DISCUSSION THREAD

**Discussion by u1550l** [AK-Homberger/NMEA2000-Workshop#4] [Q&A]
Ich habe die PGN: 127508 - Battery Status gefunden für den Status der ganze Batterie.
Mein BMS liefert die Spannungen und Temperaturen der einzelnen Zellen. Ich möchte diese über das NMEA-2000 Netzwerk senden.
Dazu habe ich keine passende PGN gefunden.
Eine Möglichkeit wäre, jede Zelle als eigenständige Batterie zu melden.
Gibt es einen besseren Vorschlag?

**Comment by AK-Homberger** [comment]
Mit den Standard-PGNs gibt es wohl keinen anderen Weg.
